contrib/azurerm/roles/generate-inventory/templates/inventory.j2

```diff
@@ -0,0 +1,33 @@
+
```

Comment:
this out of the scope of this PR. 
but ansible can take a script to generate the inventory instead of the static file. 

this is possible for example: 

``` 
ansible-playbook -i inventories/generate_my_azure_inventory.sh cluster.yaml
```
I would be nice to document some the tags/labels to apply to vms and share such generators